### ISSUE TITLE
Binary classifier

### ISSUE THREAD

**Issue by DasomKang** [DasomKang/ML#8]
1. Stochastic Gradient Descent 
- SGDClassifier in sklearn
